pkg/cloud-provider/cloudapi/internal/cloud_common.go

```diff
@@ -16,10 +16,11 @@ package internal
 
 import (
 	"fmt"
-	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sync"
 	"time"
 
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
```

Comment:
nit: no new line

Reply:
you reverted the changes and now imports is moved up under standard imports. Please move it back and remove extra line. Basically we follow standard import, third party imports and then antrea/nephe imports.